src/App.tsx

```diff
@@ -66,38 +70,42 @@ export const App: React.FC = () => {
   const [selectedMarket, setSelectedMarket] =
     useState<OneDetail['oneDetail']>();
   const [zip, setZip] = useState<string>('');
-  const [error, setError] = useState(0);
+  const [errorCode, setErrorCode] = useState<ApiMarkets["errorCode"]>("");
+  const [ loading, setLoading ] = useState<boolean>(true);
```

Comment:
We might want to default this to false and begin loading when the first fetch starts